core/testdata/testspecs/v2_specs.go

```diff
@@ -367,7 +366,7 @@ decode_log->vrf->estimate_gas->simulate
 	if vrfVersion == vrfcommon.V2Plus {
 		observationSource = fmt.Sprintf(`
 decode_log              [type=ethabidecodelog
-                         abi="RandomWordsRequested(bytes32 indexed keyHash,uint256 requestId,uint256 preSeed,uint256 indexed subId,uint16 minimumRequestConfirmations,uint32 callbackGasLimit,uint32 numWords,bool nativePayment,address indexed sender)"
+                         abi="RandomWordsRequested(bytes32 indexed keyHash,uint256 requestId,uint256 preSeed,uint256 indexed subId,uint16 minimumRequestConfirmations,uint32 callbackGasLimit,uint32 numWords,bytes extraArgs,address indexed sender)"
```

Comment:
Small nit: we could refer to `v2. RandomWordsRequestedV2PlusABI` here.

Reply:
yeah good suggestion. left it out for now due to cyclic imports but can follow up in a subsequent PR